src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/Listeners/ServiceBusEntityPathHelper.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.ServiceBus.Listeners
+{
+    internal static class ServiceBusEntityPathHelper
+    {
+        public static EntityType ParseEntityType(string entityPath)
+        {
+            return entityPath.Contains("/Subscriptions/") ? EntityType.Topic : EntityType.Queue;

```

Comment:
We need these string operations to be case insensitive, here and other places. These aren't are they?